source/process/community-guidelines.rst

```diff
@@ -0,0 +1,141 @@
+============================================================
+Mattermost Community Guidelines
+============================================================
+
+In this document we set out some easy-to-follow community guidelines, including:
+
+- `GitHub Help Wanted Issues in Platform Repository`_ - Guidelines for managing Help Wanted issues in GitHub
+- `Community Forums`_ - Guidelines for responding to `Mattermost Forum posts <forum.mattermost.org>`_
+
+----
+
+GitHub Help Wanted Issues in Platform Repository
+---------------------------------------------------------
+
+Goal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Increase Mattermost usage through more numerous and larger deployments via vibrant open source community by contributing features that would not otherwise be offered
+
+Principles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Contributors get a worthwhile benefit from their contributions from the feature directly, and/or from social recognition
+- Help Wanted issues should have a low barrier to contribute
```

Comment:
Is it possible to re-word this a bit? I know what this is trying to say, but the metaphor is not totally clear to me.